config/locales/idv/fr.yml

```diff
@@ -212,6 +211,14 @@ fr:
         arrivée, veuillez être patient car les lettres mettent jusqu’à
         <strong>10 jours</strong> pour arriver. Nous vous remercions de votre
         patience.</p>'
+      request_another_letter:
+        button: Faire une nouvelle demande
+        instructions_html: Vous pouvez demander à faire une nouvelle demande si vous
+          avez des problèmes avec votre lettre actuelle ou si elle n’est pas
+          parvenue à destination. Les lettres mettent entre <strong>5 et
+          10</strong> jours pour arriver.
+        learn_more_link: En savoir plus sur la vérification de votre adresse par courrier
+        title: Voulez-vous faire une nouvelle demande?
```

Comment:
This says, "Do you want to make a new request?" which is not the same as "Request another letter?" @artfulaction fyi

Reply:
I had the same question for the translator, but they assured me this was better. Documented here: https://docs.google.com/document/d/1mGXDY7w6LXD0BgTbkLgk9ov-743YYRasg-aTuTaHUbE/edit?disco=AAAA6C4HbSo

Reply:
Thanks for the info! Surprising, but here we are. Usage of the French version is really low according to analytics anyway.